.github/workflows/github-actions.yml

```
@@ -0,0 +1,28 @@
+name: test-deploy-gradle-sora-plugin
+on: [push]
+jobs:
+  test-deploy-plugin:
+    runs-on: ubuntu-latest
+    steps:
+      # Checks-out the repo under $GITHUB_WORKSPACE, so that workflow accesses it
+      - uses: actions/checkout@v2
+      - name: Store gradle key & secret into gradle.properties
+        env:
+          GRADLE_KEY: ${{ secrets.GRADLE_KEY }}
+          GRADLE_SECRET: ${{ secrets.GRADLE_SECRET }}
+        run: |
+          mkdir -p ~/.gradle/
+          echo gradle.publish.key=${GRADLE_KEY} >> ~/.gradle/gradle.properties
+          echo gradle.publish.secret=${GRADLE_SECRET} >> ~/.gradle/gradle.properties
+      - name: make scripts executable
+        run: |
+          chmod +x gradlew
+          chmod +x publish.sh
+      - name: Run Gradle Wrapper Daemon, Build, then Test
+        run: |
+          ./gradlew --daemon
+          ./gradlew build -x test
+          ./gradlew test
+      - name: Run Deployment Script
+        run: ./publish.sh
+        shell: bash
```

```suggestion
      - name: Run Deployment Script
        run: ./gradlew publishPlugins
        shell: bash
```
Let's move it to another workflow with:
```yaml
on:
  push:
    branches: ["master"]
```

---

Also you can delete publish since it was just wrapper with travis token

---

didn't see your comments, just was plying with that, partially fixed without knowing the warnings